### ISSUE TITLE
TamperMonkey userscript not working

### ISSUE THREAD

**Issue by Rohaq** [devunt/make-gis-great-again#62]
This was working, but it looks like Google have changed their page causing the script to fail when trying to add the button.

Example link:
https://www.google.co.uk/search?q=sega+power+strip&num=40&tbm=isch

Output from console:
```
Uncaught TypeError: Cannot set property 'innerText' of null
    at addButton (userscript.html?id=cd697653-4be5-43f7-b18b-1d1540ce687a:51)
    at addedNodes.forEach (userscript.html?id=cd697653-4be5-43f7-b18b-1d1540ce687a:81)
    at NodeList.forEach (<anonymous>)
    at mutations.forEach (userscript.html?id=cd697653-4be5-43f7-b18b-1d1540ce687a:80)
    at Array.forEach (<anonymous>)
    at MutationObserver (userscript.html?id=cd697653-4be5-43f7-b18b-1d1540ce687a:77)
```

**Comment by the3ye** [comment]
same here, stopped working recently

**Comment by Almo7aya** [comment]
It's working fine with me!!

**Comment by Brottweiler** [comment]
Same here, stopped working. Using Tampermonkey Beta `4.6.5709`.

**Comment by Rohaq** [comment]
Messes up the text alignment in the button a little, but the button is back with the above. Thanks!

**Comment by jeffrz** [comment]
Assuming they randomize the class of the span element within the <a>, it's probably not worth hardcoding. I guess you could instead delete the span:first-child and then change the text in span:last-child, but I'm not sure it's worth all of the trouble.

**Comment by devunt** [comment]
Fixed by #66.

**Comment by Brottweiler** [comment]
This seems to be a problem again, at least for me, using Tampermonkey Beta 4.6.5757, which updated today for me.
